src/mongocxx/search_index_view.cpp

```diff
@@ -65,23 +74,24 @@ std::string search_index_view::create_one(const client_session& session,
     return _get_impl().create_one(&session, model);
 }
 
-std::vector<bsoncxx::string::view_or_value> search_index_view::create_many(
+std::vector<std::string> search_index_view::create_many(
```

Comment:
I ran into some issues with reading the values of the `std::vector<bsoncxx::string::view_or_value>` returned from this function due to ownership semantics. After talking to @eramongodb, we came to the conclusion that it would be easier and more clear to just make this a `std::vector<std::string>`. This minor change seen across this PR.